shared-css.js

```diff
@@ -0,0 +1,116 @@
+import { css } from "lit";
```

Comment:
Are you adding this file twice?  I see it both at the top level and under static/sass/

Reply:
I assumed we need it in every file that uses the css literal.  The compiler (or import interpreter) ought to avoid importing something again if it has already been imported, but I don't know that it works that way.  The editor would not know about a higher-level import of css in whatever imports shared-css.js, so the editor ought to complain about css being undefined.   

I'm getting an obscure error now, after I started making more changes to add a first test, so I can't test what happens if I comment out this import.

Reply:
Removed all my extra changes, got a new package-lock.json, etc, and that fixed the obscure error.  

Now I see that commenting out the import of lit for css doesn't complain.  But nor does it complain with the import.  It seems more correct to do the import, following the principle that every file should be correct on its own.

Reply:
I was making a file-level comment on line 1 of this file rather than referring to the actual content of line 1.  Sorry for being unclear about thati.

It looks like you have mistakenly added two copies of shared-css.js to this PR: one at the top level and one under static/sass.